libraries/utilities/elasticsearch.cpp

```diff
@@ -152,9 +127,65 @@ std::vector<std::string> createBulk(std::string index_name, std::string data, st
 
    bulk.push_back("{ \"index\" : { \"_index\" : \""+index_name+"\", \"_type\" : \"data\" "+create_string+" } }");
```

Comment:
Better don't create JSON strings like this. Create a variant_object instead and let fc::json serialize it.